SecurityCourse/Controllers/AccountController.cs

```diff
@@ -33,11 +33,7 @@ public AccountController(ILogger<AccountController> logger)
         [HttpPost("[action]")]
         public AuthResponse Login([FromBody] AuthRequest request)
         {
-            var document = BsonDocument.Parse($@"{{
-                                                     Login : ""{request.Login}"" , Password: /{request.Password}/i
-                                                }}");
```

Comment:
These were leading to security vulnerability with MongoDB. The reason was unsafe bson document creation by plain string parsing, which is vulnerable to ^[\s\S]* injection and leads to password checking skip.